src/callbacks_step/analysis_dg1d.jl

```diff
@@ -1,27 +1,30 @@
 
-function create_cache_analysis(analyzer, mesh::TreeMesh,
-                               equations::AbstractEquations{1}, dg::DG, cache,
+function create_cache_analysis(analyzer, mesh::TreeMesh{1},
+                               equations, dg::DG, cache,
                                RealT, uEltype)
 
   # pre-allocate buffers
-  u_local = zeros(uEltype,
-                  nvariables(equations), nnodes(analyzer))
-  x_local = zeros(RealT,
-                  ndims(equations), nnodes(analyzer))
+  u_local = Array{uEltype}(undef,
+                           identity(nvariables(equations)), identity(nnodes(analyzer)))
```

Comment:
Resolved in 4adf208. They were remains of attempts to reduce latency. As it turned out, just removing the `inline=true` annotation from `@avx` was the main source of huge latency improvements, so that we can use the optimized versions I implemented in this PR.

Reply:
Thanks for the clarification!